src/modules/video_player/index.js

```diff
@@ -19,6 +19,30 @@ function stepPlaybackSpeed(faster) {
     currentPlayer.player.setPlaybackRate(rates[idx]);
 }
 
+const handleVisibilityChange = (function() {
+    let userMuted = $('.player-button--volume .mute-button').length;
+
+    const clickHandler = () => {
+        userMuted = $('.player-button--volume .mute-button').length;
+    };
+
+    $('.player-button--volume').on('click', clickHandler);
```

Comment:
this doesn't seem to consider the player mutating mute, so it could get inconsistent with player state .. we should rely instead on the player's mute state instead of maintaining our own

Reply:
This was rewritten